infrastructure/validators/onboarding/run-validator/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Run a Validator
+description: TODO
+hide: 
+- feedback
+template: subsection-index-page.html
+---
```

Comment:
```suggestion
---

```

Reply:
The `index.md` file need an empty line at the end to render properly

Reply:
I was going a little crazy wondering why that was happening, thank you!